api/tests/unit/features/test_unit_features_views.py

```diff
@@ -2510,6 +2511,94 @@ def test_list_features_with_intersection_tag(
     assert response.data["results"][0]["tags"] == [tag1.id, tag2.id]
 
 
+def test_list_features_with_feature_state(
```

Comment:
I think there's a few extra cases to test here (either in this test, or in a separate test): 

 1. Segment / Identity overrides are ignored
 2. The latest version is the one that is returned

Reply:
I added them into the existing test